### ISSUE TITLE
Improve help for Google Storage Classes

### ISSUE THREAD

**Issue by LukBukkit** [duplicati/duplicati#3857]
- [X] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.4.23_beta_2019-07-14
- **Operating system**: Windows 10 (1903)
- **Backend**: Google Cloud

## Description

The help information for the parameter `--gcs-storage-class` is wrong. 

Valid arguments should be: 
```
multi_regional
regional
nearline
coldline
```
Reference: https://cloud.google.com/storage/docs/storage-classes

## Steps to reproduce

1. Run `./Duplicati.CommandLine.exe help gcs`

- **Actual result**:
```
PS P:\Programme\Duplicati 2> .\Duplicati.CommandLine.exe help gcs
Google Cloud Speicher (gcs):
 Dieses Backend kann Daten von Google Cloud Storage lesen und schreiben. Das
 unterstützte Formate ist "googlecloudstore://bucket/folder".
 Unterstützte Optionen:
  --gcs-location (String): Vorgegebene Standortoption zum Erstellen eines
    Buckets
    This option is only used when creating new buckets. Use this option to
    change what region the data is stored in. Charges vary with bucket
    location. Known bucket locations:
(default):
Europe: EU
United
    States: US
Asia: ASIA
Eastern Asia-Pacific: ASIA-EAST1
Central United
    States 1: US-CENTRAL1
Central United States 2: US-CENTRAL2
Eastern
    United States 1: US-EAST1
Eastern United States 2: US-EAST2
Eastern
    United States 3: US-EAST3
Western United States: US-WEST1
  --gcs-storage-class (String): Gibt die Speicherklasse zum Erstellen eines
    Buckets an
    This option is only used when creating new buckets. Use this option to
    change what storage type the bucket has. Charges and functionality vary
    with bucket storage class. Known storage classes:
(default):
Europe:
    EU
United States: US
Asia: ASIA
Eastern Asia-Pacific:
    ASIA-EAST1
Central United States 1: US-CENTRAL1
Central United States
    2: US-CENTRAL2
Eastern United States 1: US-EAST1
Eastern United States
    2: US-EAST2
Eastern United States 3: US-EAST3
Western United States:
    US-WEST1
  --authid (Password): Der Autorisierungscode
    Das Autorisierungstoken von
    https://duplicati-oauth-handler.appspot.com?type=gcs
  --gcs-project (String): Gibt das Projekt zum Erstellen eines Buckets an
    Diese Option wird nur bei der Erzeugung eines neuen Buckets verwendet.
    Verwende diese Option, um die Projekt-ID zu liefern, mit der der Bucket
    verbunden ist. Das Projekt legt fest, wo die Nutzungsgebühren angewendet
    werden
```
- **Expected result**:
```
  --gcs-storage-class (String): Gibt die Speicherklasse zum Erstellen eines
    Buckets an
    This option is only used when creating new buckets. Use this option to
    change what storage type the bucket has. Charges and functionality vary
    with bucket storage class. Known storage classes:
(default): 
multi_regional
regional
nearline
coldline
```

**Comment by warwickmm** [comment]
Closed by #3860.  Note that the storage class options are slightly different as we are using the v1 API.

**Comment by LukBukkit** [comment]
Thanks 👍
